help/en/Acknowledgements.shtml

```diff
@@ -485,11 +485,12 @@
           LDH</a> decoders
         </li>
 
+        <li>Michael Homscheidt, who worked on the KPF-Zeller speedometer support.</li>
+
         <li>Hornby Hobbys, who provided a full set of TTS manuals plus information regarding
         various CVs.</li>
 
-        <li>Al Huberty, who organized our first users meeting at the NMRA Toronto convention
-          <a id="I"></a> <a id="J"></a>
+        <a id="I"></a> <a id="J"></a>
         </li>
```

Comment:
Is this correct? The <a> tags might be correct but the `</li>` tag seems misplaced.

Reply:
Fixed, thanks!